### PR TITLE
Feature/initial grpc

### PR DESCRIPTION
**Pull request by Bloodyhacker**

### CODE REVIEW

**Comment by Bloodyhacker** [comment]
Give it a new name.

File: parasol.iws [hunk 1]
```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (makosblade) [inline parasol.iws]: Don't want to check this file in, it should be in the .gitignore

File: parasol.ipr [hunk 2]
```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (makosblade) [inline parasol.ipr]: Don't want to check this file in, it should be in the .gitignore

**Comment by makosblade** [comment]
LGTM

File: build.gradle [hunk 3]
```diff
@@ -1,30 +1,91 @@
...
+    }
+    generateProtoTasks {
+        all()*.plugins {
+            grpc {}
+        }
+    }
+    generatedFilesBaseDir = "$projectDir/src/generated"
+}
+
+
```
Comment (makosblade) [inline build.gradle:49]: Test

**Review by makosblade — APPROVED** [review]
Test!?

File: build.gradle (hunk 3 above)
Comment (makosblade) [reply build.gradle:49]: Cool reviewing is a thing now :p

File: parasol.iws (hunk 1 above)
Comment (Bloodyhacker) [reply parasol.iws]: I see

File: parasol.ipr (hunk 2 above)
Comment (Bloodyhacker) [reply parasol.ipr]: I see

File: build.gradle (hunk 3 above)
Comment (Bloodyhacker) [reply build.gradle:49]: Nice.

**Pull request merged**